apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -339,6 +338,10 @@ open class ChatController(private val ctrl: ChatCtrl, private val ntfManager: Nt
         chatModel.updateNetworkStatus(r.contact, Chat.NetworkStatus.Connected())
 //        NtfManager.shared.notifyContactConnected(contact)
       }
+      is CR.ContactConnecting -> {
+        chatModel.updateContact(r.contact)
+        chatModel.updateNetworkStatus(r.contact, Chat.NetworkStatus.Connecting())
```

Comment:
```suggestion
```

Reply:
If we drop this we can also crop the `Connecting` object entirely